galsim/base.py

```diff
@@ -2181,6 +2181,105 @@ def getScaleRadius(self):
         return self.SBProfile.getScaleRadius()
 
 
+class Spergel(GSObject):
+    """A class describing a Spergel profile.
+
+    The Spergel surface brightness profile is characterized by three properties: its Spergel index
+    `nu`, its `flux`, and either the `half_light_radius` or `scale_radius`.  Given these properties,
```

Comment:
I thought a little more about the docstring, and i think there are 2 more (small) things I'd like to see:

* I know what a Spergel profile is and why one might want to use it, but I think that someone who hasn't read David's paper won't know that based on this docstring.  I think it would be worth having a sentence in this first paragraph saying something along the lines of the Spergel profile being Sersic-ish but with an analytic Fourier transform, so it can be faster to draw while still looking quite like a realistic galaxy profile.  This is parallel to other docstrings where we've tried to say that such-and-such class is a common form used to represent PSFs or galaxies or whatever.

* A phrase about timing considerations would be worthwhile - even just something simple, like nu=BLAH (something deVauc-ish) is roughly 3x faster to draw than a deVauc, while nu=BLAH2 (exp-ish) is similar in speed to an Exponential.

I realize you have the reference to the paper which could help address my first point, but I think someone looking at the available classes and their docstrings won't have a motivation to look at the paper and figure out what this is without some basic information that it's a profile that can describe galaxies and is sometimes faster than Sersic.